packages/flutter/lib/src/material/material.dart

```diff
@@ -35,11 +97,24 @@ class Material extends StatelessComponent {
   final int elevation;
   final Color color;
   final TextStyle textStyle;
+  final CustomMaterialPainter painter;
+
+  static MaterialInkController of(BuildContext context) {
```

Comment:
I would have expected this to return a MaterialState.  Maybe we need a different name than `of`?